### DISCUSSION TITLE
ILI6485 Serial 8Bit RGB Mode

### DISCUSSION THREAD

**Discussion by caipifrosch** [moononournation/Arduino_GFX#298] [Q&A]
Dear all, I am looking for an implementation of ILI6485 Serial 8Bit RGB Mode with following connection scheme from the ILI6485D datasheet.

Is this implemented in this library already or can anyone reference a demo project for this mode?

![image](https://github.com/moononournation/Arduino_GFX/assets/36002638/909ad16c-a629-4aeb-acc3-a769734aab3e)

![image](https://github.com/moononournation/Arduino_GFX/assets/36002638/25e32db0-b324-46b4-90c3-0a47df7c5325)

**Comment by moononournation** [comment]
Why you only connect green color pins? This library is using RGB565 color format and cannot fit your setting.

**Reply by caipifrosch** [reply]
This will give you full 24bit color while using just 8 gpio. Each pixel clocks in three 8bit values in this case.
